util/libcrypto.num

```diff
@@ -1392,7 +1392,6 @@ EVP_PBE_get                             1424	3_0_0	EXIST::FUNCTION:
 CRYPTO_nistcts128_encrypt               1425	3_0_0	EXIST::FUNCTION:
 CONF_modules_finish                     1426	3_0_0	EXIST::FUNCTION:
 BN_value_one                            1427	3_0_0	EXIST::FUNCTION:
-RSA_padding_add_SSLv23                  1428	3_0_0	EXIST::FUNCTION:DEPRECATEDIN_3_0
```

Comment:
Not sure if those should not be rather NOEXIST::FUNCTION entries as these are numbered entries and similarly below. IMO make update would do that instead of deletion.